force-app/main/default/applications/Program_Management.app-meta.xml

```diff
@@ -85,5 +112,6 @@
     <tabs>standard-Dashboard</tabs>
     <tabs>ServiceDelivery__c</tabs>
     <tabs>standard-Account</tabs>
+    <tabs>ServiceSchedule__c</tabs>
```

Comment:
Since we added tabs for the other objects, I wonder if we should just add them at the end like we talked about in CaseMan. They will end up in the dropdown for More so it wouldn't hurt anything?

Reply:
@jjbennett , I did not intentionally make them visible since during backlog grooming we said we did not want to create tabs. But we had to create them because of the icons that were displayed on the related list. We can bring this up in front of the team and we can expose the tabs.